.circleci/config.yml

```diff
@@ -27,11 +27,8 @@ jobs:
           name: Run linting
           command: npm run docker:run:lint
       - run:
-          name: Run unit tests with coverage
+          name: Run unit tests
```

Comment:
Before we were running the unit tests twice - once with coverage once without because the coverage tool was using an outdated package that caused some tests to fail. The coverage tool is up to date now and those tests no longer fail so we only need to run unit tests once